perfdash/parser.go

```diff
@@ -366,3 +366,46 @@ func parseHistogramMetric(metricName string) func(data []byte, buildNumber int,
 		}
 	}
 }
+
+func parseSystemPodMetrics(data []byte, buildNumber int, testResult *BuildData) {
+	type containerMetrics struct {
+		Name         string `json:"name"`
+		RestartCount int32  `json:"restartCount"`
+	}
+
+	type podMetrics struct {
+		Name       string             `json:"name"`
+		Containers []containerMetrics `json:"containers"`
+	}
+
+	type systemPodsMetrics struct {
+		Pods []podMetrics `json:"pods"`
+	}
+
+	build := fmt.Sprintf("%d", buildNumber)
+	var obj systemPodsMetrics
+	if err := json.Unmarshal(data, &obj); err != nil {
+		fmt.Fprintf(os.Stderr, "error parsing JSON in build %d: %v %s\n", buildNumber, err, string(data))
+		return
+	}
+
+	restartCounts := make(map[string]int32)
+
+	for _, pod := range obj.Pods {
+		for _, container := range pod.Containers {
+			restartCounts[container.Name] = container.RestartCount
+		}
+	}
+
+	perfData := perftype.DataItem{Unit: "", Labels: map[string]string{
+		"RestartCount": "RestartCount",
+	}, Data: make(map[string]float64)}
+	for container, cnt := range restartCounts {
+		curMaxCnt := float64(0)
```

Comment:
IIUC we are copying restartCounts to perfData.Data. Is it possible to have container as a key more than once?